### DISCUSSION TITLE
Not Receiving any logs

### DISCUSSION THREAD

**Discussion by hanre323** [utmstack/UTMStack#849] [Q&A]
Hi

We have a self hosted utmstack server but we are not receiving any logs, the server worked fine previously but it suddenly stopped working.

We used the installation OS.

Someone please help!

**Comment by c3s4rfred** [comment]
Hi @hanre323, can you provide more information about your issue: UTMStack's version, Host specifications, actions you've done as: network change, hardware change and so on.

Best regards
